l2geth/rollup/sync_service.go

```diff
@@ -75,8 +72,8 @@ func NewSyncService(ctx context.Context, cfg Config, txpool *core.TxPool, bc *co
 	}
 	timestampRefreshThreshold := cfg.TimestampRefreshThreshold
 	if timestampRefreshThreshold == 0 {
-		log.Info("Sanitizing timestamp refresh threshold to 15 minutes")
-		timestampRefreshThreshold = time.Minute * 15
+		log.Info("Sanitizing timestamp refresh threshold to 5 minutes")
+		timestampRefreshThreshold = time.Minute * 5
```

Comment:
Any reason why this default was changed?

Reply:
15 minutes with no timestamp updates is way too long for users. It makes batch submission slightly cheaper due to less contexts but not worth the degraded UX

Reply:
Setting it to 3 minutes https://github.com/ethereum-optimism/optimism/pull/531